### PR TITLE
Feat: List component added

### PR DESCRIPTION
**Pull request by kotalwar-krishna**
List component added.
-list releated css you can find in list.css file
**screenshots**
list1
list2
list3

### CODE REVIEW

File: components/text-utility.css [hunk 1]
```diff
@@ -54,14 +54,14 @@
     color: rgb(77, 76, 76);
 }
 
-.alignment-container{
+/* .alignment-container{
```
Comment (Kothalkarpriya) [inline components/text-utility.css]: Remove the unnecessary comments

File: documentation/doc.css [hunk 2]
```diff
@@ -143,6 +144,14 @@ html {
     align-items: stretch;
  }
 
+ .alignment-container{
+
```
Comment (Kothalkarpriya) [inline documentation/doc.css:148]: remove blank spaces

File: documentation/doc.html [hunk 3]
```diff
@@ -1,572 +1,704 @@
...
+      <!-- Badge -->
+      <div class="component-doc-container">
+        <h1 id="badge">Badge</h1>
+        <p class="component-information">
+          Use the Badge Component Symbol to draw attention to another interface element or to display a notification. It
+          enhances the component to which it is attached with additional information, disclosed to the user upon
+          interacting with it.
+        </p>
+
+
```
Comment (Kothalkarpriya) [inline documentation/doc.html]: lots of blank spaces in your code please remove it

**Review by Kothalkarpriya — COMMENTED** [review]
Please remove blank spaces and unnecessary comments

File: documentation/doc.html (hunk 3 above)
Comment (kotalwar-krishna) [reply documentation/doc.html]: ok

File: documentation/doc.css (hunk 2 above)
Comment (kotalwar-krishna) [reply documentation/doc.css:148]: ok

File: components/text-utility.css (hunk 1 above)
Comment (kotalwar-krishna) [reply components/text-utility.css]: done

**Review by Kothalkarpriya — APPROVED** [review]

**Review by sakleprajwal — COMMENTED** [review]
All good. Please go ahead

**Review by sakleprajwal — APPROVED** [review]

**Pull request merged**